### DISCUSSION TITLE
Multiple posts within a page

### DISCUSSION THREAD

**Discussion by arunism** [ppoffice/hexo-theme-icarus#1154] [Q&A]
I tried using hexo-icarus theme for my personal blog site and I noticed we usually have lists of posts at a single location(homepage by default). But I wanted to create separate pages and a number of posts related to those pages. Like pages `Tech` and `Lifestyle`. The page `Tech` has multiple posts related to technologies i come across and the page `Lifestyle` consists of multiple independent posts related to my lifestyle.

How do I figure this out? Is there any existing way that I missed out?

**Comment by ppoffice** [comment]
Add tags to your post and each tag will have an index page. Check out hexo documentation for details.

**Reply by arunism** [reply]
`/tags/tag-name/` seems ugly. I wanted something like `/page/post-title/`. Do you have any idea to do so?

**Reply by apluka34** [reply]
Hi arunism, have you found out the solution? I am struggling with the same problem. I want to create a different page named 'Daily Reading' and want to add several types of blog posts to it, and the home page should be changed to 'About me'. So how can I relocation my blog posts to other page instead of  the home page? Thank you guys so much in advance!

**Reply by arunism** [reply]
[ghimirearun.com.np ](https://ghimirearun.com.np/) This is me. Make sure if this is what you are looking for. In case it is, I will let you know how I did it.

**Reply by apluka34** [reply]
Yeah, I have contacted you via facebook. Please check the messages. Thank you so much bro.
